static/js/formatters-internal.js

```diff
@@ -560,4 +560,21 @@ export function highlightField(fieldValue, matchedSubstrings = []) {
   });
 
   return highlightedString;
+}
+
+/**
+ * Given an array of youtube videos from the KG, returns an embed link for the first video.
+ * If it is not possible to get an embed link, null is returned instead.
+ *
+ * @param {Object[]} videos 
+ * @returns {string|null}
+ */
+export function getYoutubeUrl(videos = []) {
+  if (videos.length === 0) {
+    return null;
+  }
+  const videoUrl = videos[0]?.video?.url;
```

Comment:
this stems partially from wanting green checkmarks, which might not be the best place for these desires to come from, but should we add some simple unit tests?

Reply:
ya, I'll add some tests